Modules/_ctypes/_ctypes.c

```diff
@@ -5642,6 +5642,17 @@ wstring_at(const wchar_t *ptr, int size)
     return PyUnicode_FromWideChar(ptr, ssize);
 }
 
+static PyObject *
+buffer_at(char *ptr, int size, int allow_write)
```

Comment:
Why `int size` and not `Py_ssize_t size` or `size_t size`?

Reply:
For consistency with `string_at()`

Reply:
That code is perhaps outdated. And it is expected that `buffer_at()` will be used for larger data. There should not be a drawback of supporting larger buffers.

Reply:
You are right. Since I am not a frequent contributor, I erred on the side of consistency with the surrounding code.

Reply:
Okay I made this change but I used a `Py_ssize_t` instead because that is what is passed all the way through to `PyMemoryView_FromMemory()`.